content/docs/gke/pipelines/authentication-pipelines.md

```diff
@@ -9,49 +9,117 @@ This page describes authentication for Kubeflow Pipelines to GCP.
-The following helper bash scripts bind workload identities for KSAs provided by Pipelines Standalone:
+##### 2. Deploy Kubeflow Pipelines
+Deploy via [Pipelines Standalone](/docs/pipelines/installation/overview/#kubeflow-pipelines-standalone) or [GCP hosted ML pipelines](/docs/pipelines/installation/overview#gcp-hosted-ml-pipelines) as usual.
+
+Note, for GCP hosted ML pipelines additionally, you need to patch `proxy-agent` to work with workload identity:
+```
+kubectl patch deployment proxy-agent --patch '{"spec": {"template": {"spec": {"hostNetwork": true}}}}'
+```
+
+##### 3. Bind Workload Identities for KSAs used by Kubeflow Pipelines
+
+The following helper bash scripts bind Workload Identities for KSAs used by Kubeflow Pipelines:
 
 * [gcp-workload-identity-setup.sh](https://github.com/kubeflow/pipelines/blob/master/manifests/kustomize/gcp-workload-identity-setup.sh) helps you create GSAs and bind them to KSAs used by pipelines workloads. This script provides an interactive command line dialog with explanation messages.
-* [wi-utils.sh](https://github.com/kubeflow/pipelines/blob/master/manifests/kustomize/wi-utils.sh) provides minimal utility bash functions that let you customize your setup. The minimal utilities make it easy to read and use programmatically.
+* [wi-utils.sh](https://github.com/kubeflow/pipelines/blob/master/manifests/kustomize/wi-utils.sh) alternatively provides minimal utility bash functions that let you customize your setup. The minimal utilities make it easy to read and use programmatically.
+
+##### 4. Configure IAM permissions of used GSAs
+
+* Pipelines use `pipeline-runner` KSA. Configure IAM permissions of the GSA bound to this KSA to allow pipelines use GCP APIs.
+
+* Pipelines UI uses `ml-pipeline-ui` KSA. Pipelines Visualization Server uses `ml-pipeline-visualizationserver` KSA. If you need to view visualizations stored in Google Cloud Storage (GCS) from pipelines UI, you should add Storage Object Viewer permission (or the minimal required permission) to their bound GSA.
+
+#### Cluster setup to use Workload Identity for Full Kubeflow
 
-Pipelines use `pipeline-runner` KSA, you can configure IAM permissions of the GSA bound to this KSA to allow pipelines use GCP APIs.
+Since Kubeflow 1.0, if you deployed Kubeflow following the GCP instructions Workload Identity has already been set up correctly for Kubeflow Pipelines.
 
-Pipelines UI uses `ml-pipeline-ui` KSA. If you need to view visualizations stored in Google Cloud Storage (GCS) from pipelines UI, you should add Storage Object Viewer permission to its bound GSA.
+Pipelines use `kf-user` KSA by default which is different from Kubeflow Standalone. You can also choose to use a different default KSA by overriding deployment like [this overlay](https://github.com/kubeflow/manifests/blob/a8d473a2431bd7afb88834f62751b021c4269817/pipeline/api-service/overlays/use-kf-user/deployment.yaml#L12).
```

Comment:
Would it make sense to add more instructions to install the overlay?

Reply:
The overlay will be installed by default. I just want to point out where it is configured, so it can be customized if the user wants.

Do you have a suggestion on how to make that clearer? Or maybe we can just skip the customizing overlay part?

Reply:
I think it would make sense to skip it, or to direct users to the [gke/authentication](https://www.kubeflow.org/docs/gke/authentication/) doc.

Also, why is the KSA different between standalone and full KF?

Reply:
It's a purely tech related tradeoff. Ideally, when using workload identity, we should bind existing KSAs to GSAs based on requirement. That's what we did with standalone deployment too.

But Kubeflow's kfctl doesn't have good support for flexible workload identity binding, the workaround we come to a conclusion now is to use a pre bound KSA `kf-user`: https://github.com/kubeflow/kubeflow/issues/4803#issuecomment-592306016

Reply:
Updated doc